components/molecules/Pagination/pagination.tsx

```diff
@@ -121,7 +116,7 @@ const Pagination = ({
         )}
         {goToPage && (
           <div className="hidden md:block">
-            <PaginationGotoPage totalPage={totalPage} page={page} setPage={handleSelected} name={""} />
+            <PaginationGotoPage totalPage={totalPage} page={currentPage} setPage={handleSelected} name={""} />
```

Comment:
Definitely not for the PR, but the name prop makes no sense here as this should be static. we can name the input directly in https://github.com/sourabpramanik/open-sauced-app/blob/a3c9d2ad460fcefe7c4607ccf96773dcf6e73514/components/molecules/PaginationGotoPage/pagination-goto-page.tsx#L33
and remove the name prop. I'll open a ticket for this tiny refactor 👍

Reply:
Sure will remove that

Reply:
@OgDev-01 did you raised the ticket for this refractor?